tests/conftest.py

```diff
@@ -88,24 +105,70 @@ def mock_server():
 
 @pytest.fixture
 def live_mock_server(request):
-    from tests.mock_server import create_app
-    if request.node.get_closest_marker('port'):
-        port = request.node.get_closest_marker('port').args[0]
-    else:
-        port = 8765
-    app = create_app(utils.default_ctx())
-    server = Process(target=app.run, kwargs={"port": port, "debug": True,
-                                             "use_reloader": False})
-    server.start()
+    port = utils.free_port()
+    #  TODO: Windows can't pickle
+    #  app = utils.create_app(utils.default_ctx())
+    #  def worker(app, port):
+    #    app.run(host="localhost", port=port, use_reloader=False, threaded=True)
+    #  server = Process(target=worker, args=(app, port))
+    #  server.start()
+    root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+    path = os.path.join(root, "tests", "utils", "mock_server.py")
+    command = ["python", path]
+    env = os.environ
+    env["PORT"] = str(port)
+    env["PYTHONPATH"] = root
+    server = subprocess.Popen(command, env=env)
+    server.base_url = "http://localhost:%s" % port
```

Comment:
Windows couldn't run the process with multiprocessing, so I just launch a new process.  There's substantial overhead here, we should launch one server for all tests and make it maintain state by test.